ert3/engine/_run.py

```diff
@@ -74,11 +76,13 @@ def _transmitter_map_resources(
     futures: List[TransmitterCoroutine] = []
     for input_ in inputs:
         file_path = workspace.get_resources_dir() / input_.source_location
-        collection = ert.data.load_collection_from_file(
-            file_path,
-            input_.source_mime,
-            ensemble_size=ensemble_size,
-            is_directory=input_.source_is_directory,
+        collection = get_event_loop().run_until_complete(
```

Comment:
Why not join this work with what's going into the future to be asynchronously executed anyway?

Reply:
A pattern that is used is an inline async function with two await in order.

Reply:
Thx, will try

Reply:
This expanded into also making `load_record` an async function..

Reply:
Perfect.